wait_for/__init__.py

```diff
@@ -89,6 +90,25 @@ def _get_failcondition_check(fail_condition):
         return partial(check_result_is_fail_condition, fail_condition)
 
 
+def _get_handled_exceptions(handle: Union[Type[Exception], Iterable[Type[Exception]]]) \
+        -> Iterable[Type[Exception]]:
+    if isinstance(handle, type) and issubclass(handle, Exception):
+        return iter((handle,))
+    else:
+        if isinstance(handle, Iterable):
+            return iter(handle)
+        else:
+            return iter((Exception,))
+
+
+def _check_must_be_handled(exception: Exception,
+                           handle: Union[Type[Exception], Iterable[Type[Exception]]]) \
```

Comment:
I think `bool` should be part of the typing Union here as well.

Reply:
I would like to give the possibility to the user to pass any value which could be cast to True, not just True or False